memdb.go

```diff
@@ -137,7 +137,7 @@ func (db *MemDB) DeleteSync(key []byte) error {
 }
 
 // Close implements DB.
-func (db *MemDB) Close() error {
+func (*MemDB) Close() error {
```

Comment:
I think lint will complain that the signature is not following the same pattern for all functions. It's probably a style preference so not a strong opinion but it usually complains in case of mixed `(db *MemDB)` and `(*MemDB)`.

Reply:
Ah, nope, please think of an unused receiver as the declaration of an unused variable, and these lints will make perfect sense to you.